### ISSUE TITLE
Disk/Washer Exercise Graph Labels

### ISSUE THREAD

**Issue by grady** [APEXCalculus/APEXCalculusPTX#161]
In Section 7.2 on Volumes using Disk/Washer method. In many problems, the equation of the curve to be revolved are shown inside the plot. The positioning of the function equation label needs adjusting in many cases. In particular _problem 4 and 8 have the power of 2 completely cut off,_ displaying a linear equation for what is obviously a quadratic curve.

Even better than adjusting the position of the equation label in the plot, I think the equation and bounds of the region to be revolved should be given in text. This would be better from an accessibility perspective as well.

https://opentext.uleth.ca/apex-video/sec_disk.html

**Comment by grady** [comment]
This problem also occurs in the exercises for the Shell Method in the next section.

**Comment by Alex-Jordan** [comment]
One could try to fix these by changing how cropping works, so that it would now crop to encompass all label text too. This would have the side effect of making adjacent or nearby images seem misaligned. Visually, the axis windows would differ each time a label extended outside of the axis window.

One could try to fix these by visiting each image and adjusting the positioning of these labels until they are within the axis window. That would be tedious and prone to error. It could still cause problems in the future if there is something like a font size change, which might move something to once again be cropped or otherwise mess with carefully laid out positioning.

One could try to fix these by using an `\addlegendentry` to label these curves instead of putting node labels directly on the image somewhere. This would also be tedious. More tedious than above, because you would probably want to do this for all images for the sake of consistent styling. So not just the currently misbehaving images. But I think this is what I lean toward out of the options I have considered. 

Are there other ideas? Or a different preference among these three?

**Comment by grady** [comment]
I think the function definition should be presented in the text, not in the images. Something along the lines of 

> Find the volume enclosed when the curve $f(x)=3-x^2$ is revolved about the $x$-axis.

with the graph attached as a visual aid.

**Comment by sean-fitzpatrick** [comment]
OK, back at work. But I guess I will be focused on things like "teaching my students" and "salvaging the semester", so unlikely to turn my attention to this any time soon.

Seems like this is a bigger project that should be systematically addressed:

1. Try Alex's suggestion of adding `\addlegendentry` to the styling for labels throughout. (Can this be done in global styling?) Presumably we should test to make sure this solves the problem before proceeding ;-)
2. Identify all the places in the book where there is information in an image that should be repeated in the text of the exercise.

So unlikely this gets done in a weekend. Long term, we need to address the lack of alt text descriptions on our images. I'm not sure that we have any funds I could apply to to hire a student for this.

**Comment by sean-fitzpatrick** [comment]
Right now, we have the following in the `docinfo` to control styling for all the pgf plots:
```
\pgfplotsset{every axis/.append style = {
      cycle list name = curvestylelist,
      %tick label style = {font = \scriptsize},
      axis x line = middle,
      axis y line = middle,
      xlabel = {$x$},
      ylabel = {$y$},
      %x label style = {font = \scriptsize},
      %y label style = {font = \scriptsize},
      minor x tick num = 1,
      minor y tick num = 1,
      %width = {ifthenelse(.9\linewidth>240pt,240pt,.9\linewidth)},
      name=myplot,
    }}
```
Note the label style commands that are commented out: throughout the book, text in TikZ images uses `\scriptsize`. This dates back to the original LaTeX by @APEXCalculus. I thought I would try doing this in the global styling rather than in each image, but it didn't work.

Probably this is because my TikZ expertise is very limited. (Some might say non-existent.)
Most text labels are done as a `node`. Maybe `node` is not the same as `label` and this is why the styling had no effect.

But the presence of `\scriptsize` whenever there is text in an image might make a global find+replace possible?
@Alex-Jordan if you have any suggestions, let me know. Otherwise I can tinker a bit.

**Comment by Alex-Jordan** [comment]
I think that clip=true is used in the pgfplots settings. This prevents
adjacent images from seeming misaligned. But is what cuts things off before
the pdf cropping. If it were me, I'd seek a careful understanding of
everything before investing time in editing a lot of images source. To my
mind, the addlegendentry solution feels best. But I still don't understand
everything I'd like to.

**Comment by sean-fitzpatrick** [comment]
OK -- looking a bit more carefully now.

For the first of the exercises referenced at the start of the issue, we have the following code in source:
```
\addplot [firstcurvestyle,areastyle,domain=-1.73:1.73] {3-x^2};
\addplot [firstcurvestyle,-,domain=-1.73:1.73,samples=50] {3-x^2} node [shift={(-5pt,80pt)} ,black] { $y=3-x^2$};
```

Interesting to note that there was some previous awareness of label placement, given the `shift` attribute on the node.
The problem is definitely over-cropping in this case: if I build this on its own, the exponent makes it into the graph. So we're losing it at the cropping stage. (Maybe a side-effect of the change in cropping software?)

So, we can delete the node, and then add the line `\addlegendentry{$y=3-x^2$}`.

But I can't figure out how to avoid the "legend" part of the legend entry and keep only the text:
file:///home/sean/Pictures/Screenshots/Screenshot%20from%202022-05-09%2015-48-56.png

**Comment by sean-fitzpatrick** [comment]
Resolved in #192
